src/main/java/com/backbase/billpay/fiserv/payments/PaymentsServiceImpl.java

```diff
@@ -103,8 +105,10 @@ public void deletePaymentById(Header header, String id) {
     }
 
     @Override
-    public BillPayPaymentsGetResponseBody getBillPayPayments(Header header, String status, Date startDate, Date endDate,
+    public BillPayPaymentsGetResponseBody getBillPayPayments(Header header, String status, LocalDate sDate, LocalDate eDate,
                     String payeeId, String paymentType, Integer from, Integer size, String orderBy, String direction) {
+        Date startDate = fromLocalDate(sDate);
```

Comment:
Would be better to keep these as LocalDates and change the numberOfDays calculation to use LocalDates instead as it's only using the date aspect, time is not required.